config/webhook/service.yaml

```diff
@@ -17,4 +17,4 @@ spec:
       protocol: TCP
       targetPort: 9443
   selector:
-    control-plane: glance-controller-manager
+    app.kubernetes.io/name: glance-operator
```

Comment:
This alteration to the default `operator-sdk` scaffolding is what allows webhook traffic to be properly routed and not clash with other operators.